contracts/interfaces/IveERC20v2.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/token/ERC20/ERC20Upgradeable.sol";
+
+interface IveERC20v2 {
+    /* ====================== VARS ====================== */
+    function alreadyWithdrawn (address user) external view returns (uint256 amountAlreadyWithdrawn);
+
+    /* ===================== EVENTS ===================== */
+    event Claimed(address account, uint256 claimAmount);
+
+    /* ===================== ERRORS ===================== */
+
+    error TGEInPastError();
```

Comment:
TGEInPastError, lets keep all names with abbrivaiation with only first letter uppercase for readability